Remove .NET Core "LTS" mentions (#108357)
Azure/azure-content: articles/azure-monitor/app/worker-service.md

Specific instructions for each type of application are described in the following sections.

## .NET Core Worker Service application

The full example is shared at the [NuGet website](https://github.com/microsoft/ApplicationInsights-dotnet/tree/develop/examples/WorkerService).

> 1. [Download and install .NET Core](https://dotnet.microsoft.com/download).

Reviewer: ```suggestion
1. [Download and install the .NET SDK](https://dotnet.microsoft.com/download).
```

1. Create a new Worker Service project either by using a Visual Studio new project template or the command line `dotnet new worker`.

> 1. Install the [Microsoft.ApplicationInsights.WorkerService](https://www.nuget.org/packages/Microsoft.ApplicationInsights.WorkerService) package to the application.

Reviewer: ```suggestion
1. Add the [Microsoft.ApplicationInsights.WorkerService](https://www.nuget.org/packages/Microsoft.ApplicationInsights.WorkerService) package to the application.
```

    ```

1. Set up the connection string.

>    Use the same `appsettings.json` from the preceding [.NET Core](/fundamentals/) Worker Service example.

Reviewer: ```suggestion
   Use the same `appsettings.json` from the preceding [.NET](/fundamentals/) Worker Service example.
```

## .NET Core/.NET Framework console application

> As mentioned in the beginning of this article, the new package can be used to enable Application Insights telemetry from even a regular console application. This package targets [`NetStandard2.0`](/dotnet/standard/net-standard), so it can be used for console apps in [.NET Core](https://learn.microsoft.com/dotnet/fundamentals/) or higher, and [.NET Framework](https://learn.microsoft.com/dotnet/framework/) or higher.

Reviewer 2: ```suggestion
As mentioned in the beginning of this article, the new package can be used to enable Application Insights telemetry from even a regular console application. This package targets [`.NET Standard 2.0`](/dotnet/standard/net-standard), so it can be used for console apps in [.NET Core](https://learn.microsoft.com/dotnet/fundamentals/) or higher, and [.NET Framework](https://learn.microsoft.com/dotnet/framework/) or higher.
```
don't know if you want to fix any of those ^^

Reviewer 2: fixed

### EventCounter

> `EventCounterCollectionModule` is enabled by default, and it will collect a default set of counters from [.NET Core](/fundamentals/) apps. The [EventCounter](eventcounters.md) tutorial lists the default set of counters collected. It also has instructions on how to customize the list.

Reviewer: ```suggestion
`EventCounterCollectionModule` is enabled by default, and it will collect a default set of counters from [.NET](/fundamentals/) apps. The [EventCounter](eventcounters.md) tutorial lists the default set of counters collected. It also has instructions on how to customize the list.
```

### Can I enable Application Insights monitoring by using tools like Azure Monitor Application Insights Agent (formerly Status Monitor v2)?

> No. [Azure Monitor Application Insights Agent](./application-insights-asp-net-agent.md) currently supports [.NET Core](/fundamentals/) only.

Reviewer: ```suggestion
No. [Azure Monitor Application Insights Agent](./application-insights-asp-net-agent.md) currently supports [.NET](/fundamentals/) only.
```

Use this sample if you're in ASP.NET Core and creating background tasks in accordance with [official guidance](/aspnet/core/fundamentals/host/hosted-services).

[.NET Core Worker Service](https://github.com/microsoft/ApplicationInsights-dotnet/tree/develop/examples/WorkerService):

> Use this sample if you have a [.NET Core](/fundamentals/) Worker Service application in accordance with [official guidance](/aspnet/core/fundamentals/host/hosted-services?tabs=visual-studio#worker-service-template).

Reviewer: ```suggestion
Use this sample if you have a [.NET](/fundamentals/) Worker Service application in accordance with [official guidance](/aspnet/core/fundamentals/host/hosted-services?tabs=visual-studio#worker-service-template).
```
